pytest_fixtures/satellite_auth.py

```diff
@@ -145,3 +146,60 @@ def enroll_idm_and_configure_external_auth():
         f"--server {settings.ipa.hostname_ipa} "
         f"--realm {domain.upper()} -U"
     )
+
+
+@fixture(scope='session')
+def enroll_ad_and_configure_external_auth():
+    """Enroll the Satellite6 Server to an AD Server."""
```

Comment:
```suggestion
    """Enrol Satellite Server to an AD Server."""
```
No need for 'the' as Satellite Server is the product name.